src/6model/reprs/MultiDimArray.c

```diff
@@ -459,6 +459,9 @@ static void unshift(MVMThreadContext *tc, MVMSTable *st, MVMObject *root, void *
 static void shift(MVMThreadContext *tc, MVMSTable *st, MVMObject *root, void *data, MVMRegister *value, MVMuint16 kind) {
     MVM_exception_throw_adhoc(tc, "Cannot shift a fixed dimension array");
 }
+static void aslice(MVMThreadContext *tc, MVMSTable *st, MVMObject *src, void *data, MVMObject *dest, MVMint64 start, MVMint64 end) {
+    MVM_exception_throw_adhoc(tc, "Multidim slice not yet implemented");
```

Comment:
I don't think the given signature makes sense for multidim arrays, so maybe it should die like asplice just below?